docs/platform/howto/byoc/create-custom-cloud.md

```diff
@@ -822,6 +859,13 @@ Your new custom cloud is ready to use only after its status changes to
    --account "ACCOUNT_IDENTIFIER"
    ```
 
+   where
+
+   - `organization-id` is the identifier of your Aiven organization that is to be
+     connected with your own cloud account to create the custom cloud.
+   - `byoc-id` is the identifier of your custom cloud.
+   - `account` is the identifier of your account (organizational unit) in Aiven.
```

Comment:
Example: `a484338c34d7`
I hope the customer knows how to get this value.

Reply:
Recommended using CLI command `avn organization list` to extract the vaule